README.md

```diff
@@ -55,6 +55,25 @@ Images are customizable, default ones are used when no image is provided.
     NSLog(@"%0.2f - %0.2f", slider.leftValue, slider.rightValue);
 }
 
+//For Swift 3
+func rangeSliderValueDidChange(_ slider: MARKRangeSlider) {
+        print("right \(slider.rightValue) left \(slider.leftValue)")
```

Comment:
Sorry what do you mean with fix indentation? My code doesn't fix anything. It should be just an example for Swift 3